modules/indexing/src/main/java/org/apache/ignite/internal/visor/verify/ValidateIndexesClosure.java

```diff
@@ -315,7 +320,7 @@ private VisorValidateIndexesJobResult call0() {
         try {
             AtomicBoolean cpFlag = new AtomicBoolean();
 
-            if (db instanceof GridCacheDatabaseSharedManager) {
+            if (checkCrc && db instanceof GridCacheDatabaseSharedManager) {
```

Comment:
1. why do we need this check here? 
2. this check double checks near, is it correct?

Reply:
question still  actual.

Reply:
I've removed the second checkCrc condition inside the higher condition. So, double check is resolved. I hope the issue is solved.